pmm/pmm2-client-autobuild-arm.groovy

```diff
@@ -69,6 +69,7 @@ pipeline {
                     steps {
                         sh './build/bin/build-client-rpm centos:7'
                         sh './build/bin/build-client-rpm rockylinux:8'
+                        sh './build/bin/build-client-rpm almalinux:9.0'
```

Comment:
Is there a reason why we're using a different Red Hat derivative for each major version?
Would it make sense to consolidate this to just using Oracle Linux, for example?

Reply:
@LenzGr Oracle Linux doesn't have the final 9.0 version yet. 9.0 preview was released a couple of days ago.

Rocky also was chosen before we decided to use Oracle Linux.

We use these images only for building and I think it'll be easy to migrate to OL later.

Reply:
@nikita-b 
+ for @LenzGr, why not to use RHEL9 AMI for builds and ubi9 for doker?

Reply:
one more +1 to comments above

Reply:
@LenzGr as @vorsel mentioned ubi9 is available

Reply:
@vorsel @EvgeniyPatlan @LenzGr For UBI we need to change our build process. It's the same for RHEL7-9 now but for UBI we'll need to change some things. For example, we use EPEL repos and we'll need to install it in another way.

I agree that it'll be cool to use UBI everywhere but it seems a good task for a separate ticket.

Also, RHEL on AWS is not free: https://aws.amazon.com/partners/redhat/faqs/
We'll need to pay additional money to Redhut or use a developer license (it's additional bureaucracy)

We use these images for testing purposes only (we just install our package) and I think it's not worth additional resources :)

Reply:
It is not for testing! Packages will go to prod